### PR TITLE
Permit sending to EDDN from released beta builds

### PR DESCRIPTION
**Pull request by Tkael**

### CODE REVIEW

File: EDDI/EDDI.cs [hunk 1]
```diff
@@ -238,11 +238,11 @@ void ActionUpdateSquadronSystem()
 
         public bool ShouldUseTestEndpoints()
         {
-#if DEBUG
-            return true;
+#if RELEASE
```
Comment (richardbuckle) [inline EDDI/EDDI.cs]: Bzzzt. There is no RELEASE symbol!

**Review by richardbuckle — CHANGES_REQUESTED** [review]
This one is important.

File: EDDI/EDDI.cs (hunk 1 above)
Comment (Tkael) [reply EDDI/EDDI.cs]: I am fairly reminded that there is no RELEASE preprocessor symbol (unless we define one, which we don't want to do)

**Review by richardbuckle — APPROVED** [review]

**Comment by srguglielmo** [comment]
Ah! I was wondering why EDDN wasn't updating when I visited stations anymore. I had upgraded to the beta release to fix some recent bugs.

If this PR is not desired, I'd suggest adding a warning text somewhere (maybe on the EDDN Responder tab).

Thank you for all the work you do on EDDI!

**Pull request merged**